weave/trace_server/trace_server_interface.py

```diff
@@ -377,6 +377,9 @@ class ObjectVersionFilter(BaseModel):
 class ObjQueryReq(BaseModel):
     project_id: str
     filter: Optional[ObjectVersionFilter] = None
+    # Defaults to true, pass in False for a more performant
+    # query that does not return the object value
+    include_val_dump: bool = True
```

Comment:
"dump" is an internal detail - users think of this as `val`.

I think the `include_*` pattern is used to include something that does not normally come for free. I think instead we might want to use the `column` pattern from calls to allow a sub selection of fields.

Reply:
Also, tests (:

Reply:
yes yes tests, first making sure you think the approach is sound. ill play with the request